pkg/envoy/ads/profile.go

```diff
@@ -29,23 +28,23 @@ func xdsPathTimeTrack(startedAt time.Time, typeURI envoy.TypeURI, proxy *envoy.P
 		Observe(elapsed.Seconds())
 }
 
-func (s *Server) trackXDSLog(cn certificate.CommonName, typeURL envoy.TypeURI) {
+func (s *Server) trackXDSLog(proxyID string, typeURL envoy.TypeURI) {
```

Comment:
If the proxy's "name" is what's being passed here, I'd rename this accordingly since the proxy also has a plain "(UU)ID" which might make this confusing.

Reply:
Good call! Note that I'm only having a name, to maintain the usefulness of the CommonName which included both the service identity and uuid, to ensure that all the information is included for customers.

I'd rather just key it by UUID, but not sure if that reduces readability/usefulness. Curious on your thoughts here. Adding @shashankram to comment as well

Reply:
In the past, logging by UUID was rather inconvenient to debug the concerned proxy by looking at the logs. Since the CommonName used k8s primitives such as service account and namespace, it made parsing these logs easier. So I would say let's continue to add additional context in the logs and not just its UUID.

Reply:
yup, that's what i figured. Modified the field name!

Reply:
Yeah, I figured if this is ultimately shown as debug info then including some more meaningful info like the service name/namespace is valuable here. Figuring out what's what based only on UUIDs would be a lot more involved.